### DISCUSSION TITLE
Django ORM with Airflow

### DISCUSSION THREAD

**Discussion by NonlinearNimesh** [apache/airflow#26655] [Q&A]
Hi I am trying to use Django ORM with Airflow to build my project. I research a lot and found out conventionally it is not possible. Found one link (https://stackoverflow.com/questions/47470245/django-orm-in-airflow-is-it-possible) where they have mentioned it is kindoff possible but i am not able to setup my project. The database i am using is Mysql. Without Airflow i just have to mention the user, ip, port and password to connect db with my project.

Can anyone please help me how i can connect the DB.

Thanks

**Comment by potiuk** [comment]
I do not know if there is anything specific with Django ORM to connect with Airflow. Airflow uses SQLAlchemy but if you want to install django and use some of the ORM of Django to do some other stuff in your DAGs, this should be possible (as long as you can match dependencies). Airflow DAGs are Python code and as long as you avoid using top-level code connecting to DB (See best practices in the docs) you can do anything you'd do in Python script. It's entrely up to you,
